### ISSUE TITLE
builtins.Exception: Consumer asked us to stop producing

### ISSUE THREAD

**Issue by realitygaps** [matrix-org/synapse#4661]
### Description

Upgraded to 0.99.1.1 , now homeserver seems to refuse not receive/send messages to/from other servers after some time - richdvh asked to post logs here

```2019-02-17 18:01:14,868 - synapse.http.site - 203 - WARNING - GET-231208- Error processing request <SynapseRequest at 0x7f1af786ffd0 method='GET' uri='/_matrix/media/v1/thumbnail/olinix.ddnss.de/okmeMRNAFMmnRJRStjAJcGvB?width=14&height=14&method=crop' clientproto='HTTP/1.0' site=8008>: <class 'twisted.internet.error.ConnectionDone'> Connection was closed cleanly.
2019-02-17 18:01:15,107 - synapse.http.server - 112 - ERROR - GET-231206- Failed handle request via <function ThumbnailResource._async_render_GET at 0x7f1b3b7e9ae8>: <SynapseRequest at 0x7f1aeff6c2b0 method='GET' uri='/_matrix/media/v1/thumbnail/matrix.org/qOyrKnGkzhfiCgemvposcyLL?width=14&height=14&method=crop' clientproto='HTTP/1.0' site=8008>: Traceback (most recent call last):
--- <exception caught here> ---
  File "/opt/venvs/matrix-synapse/lib/python3.5/site-packages/synapse/http/server.py", line 81, in wrapped_request_handler
    yield h(self, request)
  File "/opt/venvs/matrix-synapse/lib/python3.5/site-packages/synapse/rest/media/v1/thumbnail_resource.py", line 83, in _async_render_GET
    width, height, method, m_type
  File "/opt/venvs/matrix-synapse/lib/python3.5/site-packages/synapse/rest/media/v1/thumbnail_resource.py", line 259, in _respond_remote_thumbnail
    yield respond_with_responder(request, responder, t_type, t_length)
  File "/opt/venvs/matrix-synapse/lib/python3.5/site-packages/synapse/rest/media/v1/_base.py", line 137, in respond_with_responder
    yield responder.write_to_consumer(request)
builtins.Exception: Consumer asked us to stop producing
```

### Steps to reproduce

Upgraded to 0.99.1.1 , now homeserver seems to refuse not receive/send messages to/from other servers after some time - richdvh asked to post logs here

### Version information

- **Version**: 0.99.1.1

- **Install method**: 
debs from matrix.org

- **Platform**: 
debian 9 in a kvm

**Comment by richvdh** [comment]
Thanks for the log; though I think the error is harmless and unrelated to your federation problems (note that it's in response to a client request for an image thumbnail)

**Comment by neilisfragile** [comment]
@realitygaps it sounds like the logs are benign - logs aside are you reporting that you are unable to federate on 0.99.1.1?

**Comment by richvdh** [comment]
fixed in #4738
